presto-cli/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "resources":[
+    {"pattern":"\\QMETA-INF/services/org.jline.terminal.spi.JnaSupport\\E"},
+    {"pattern":"\\Qcom/sun/jna/darwin/libjnidispatch.jnilib\\E"},
+    {"pattern":"\\Qorg/jline/utils/capabilities.txt\\E"},
+    {"pattern":"\\Qorg/jline/utils/xterm-256color.caps\\E"},
+    {"pattern": "org/jline/utils/.*caps$"},
+    {"pattern": "org/jline/utils/capabilities\\.txt$"},
```

Comment:
These look wrong. We don’t use AWS or Jansi.

Reply:
this was autocollected by agent, probably needs cleanup after I switched back to jline-jna from jline-jansi